physionet-django/physionet/settings/base.py

```diff
@@ -107,6 +113,15 @@
 # Session management
 
 SESSION_COOKIE_SECURE = True
+SESSION_EXPIRE_AT_BROWSER_CLOSE = True
+
+# Idle timeout
+SESSION_SECURITY_WARN_AFTER = config('SESSION_SECURITY_WARN_AFTER', cast=int, default=540)
+SESSION_SECURITY_EXPIRE_AFTER = config('SESSION_SECURITY_EXPIRE_AFTER', cast=int, default=600)
```

Comment:
I don't think we need this either, right? And also I don't think we need the default values for this. It has default values set in https://github.com/yourlabs/django-session-security/blob/master/session_security/settings.py

Reply:
Check whether it will work though, we don't want to get into any not-defined vs. blank value shenanigans.